providers/aws/provider/provider.go

```diff
@@ -13,16 +13,12 @@ import (
 	"go.mondoo.com/cnquery/providers-sdk/v1/upstream"
 	"go.mondoo.com/cnquery/providers/aws/connection"
 	"go.mondoo.com/cnquery/providers/aws/resources"
-	osconnection "go.mondoo.com/cnquery/providers/os/connection"
 	"go.mondoo.com/cnquery/providers/os/connection/shared"
-	"go.mondoo.com/cnquery/providers/os/detector"
 )
 
 const (
-	defaultConnection           uint32 = 1
-	DefaultConnectionType              = "aws"
-	SshConnectionType                  = "ssh"
-	RegistryImageConnectionType        = "registry-image"
```

Comment:
This removal is also part of https://github.com/mondoohq/cnquery/pull/1707.

But I removed even more here.